test/Identity.IntegrationTest/Endpoints/IdentityServerTests.cs

```diff
@@ -442,6 +546,29 @@ Task<HttpContext> MakeRequest()
         }
     }
 
+    private async Task CreateOrganizationWithSsoPolicy(string username, bool ssoPolicyEnabled)
+    {
+        var userRepository = _factory.Services.GetService<IUserRepository>();
+        var organizationRepository = _factory.Services.GetService<IOrganizationRepository>();
+        var organizationUserRepository = _factory.Services.GetService<IOrganizationUserRepository>();
+        var policyRepository = _factory.Services.GetService<IPolicyRepository>();
+
+        var organization = new Bit.Core.Entities.Organization { UseSso = true };
+        await organizationRepository.CreateAsync(organization);
+
+        var user = await userRepository.GetByEmailAsync(username);
+        var organizationUser = new Bit.Core.Entities.OrganizationUser
+        {
+            OrganizationId = organization.Id,
+            UserId = user.Id,
+            Status = OrganizationUserStatusType.Confirmed
+        };
```

Comment:
Don't we have to make them an Owner to test this properly? This makes me wonder why/how the tests are passing at the moment. I might be missing something though.

Reply:
The tests were passing because `Owner` is the default value in the enum `OrganizationUserType.Owner = 0`. I've added tests for all types of users.

Reply:
Thanks for explaining - I missed that default value. Better to be explicit I think, and good job adding tests for other roles as well.